README.md

```diff
@@ -1,26 +1,27 @@
-# Math-magicians
+# Math magicians
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-![math-magicians_]()
+![math-magicians_](./src/images/math_magicians.png)
 
 ## Built With
 
 This project is build with:
```

Comment:
## [OPTIONAL] kindly you have a typing error here. write instead `This project is built with:`